test/games/strategy/triplea/delegate/GameDataTestUtil.java

```diff
@@ -80,6 +80,27 @@ public static UnitType carrier(final GameData data) {
     return unitType(Constants.UNIT_TYPE_CARRIER, data);
   }
 
+  // Some units hard coded here rather than placed in Constants at the request of community members Ref: Pull Request 1074
```

Comment:
> "I would have thought it would be consistent with the rest of the system for it to be defined there."

I think you raise a good point. But, the pattern is not very good since we have test only code in production code. Consistency to a bad pattern sometimes can be better than inconsistency, in this case the inconsistency has a number of benefits (most significant being the split of test/prod code) 

--- -
RE: line length
120 is the limit per Veqryn. Until not long ago (9 months ago) I actually had a screen that only rendered 90 characters wide comfortably (was a very nice upgrade to a 2k monitor) . 
One benefit is reducing horizontal scrolling. Code is often viewed in console windows, or web browsers, or side by side, and in many contexts really long lines cannot be seen fully. 

It is also to encourage lines that are not super long. Part of the trick is that the indentation should not be very deep, only about 6 or 8 characters from the left, this gives 72-74 characters, or in our case 112 to 114 characters per line - which is a good bit of code for any one line.